input/input.h

```diff
@@ -56,6 +56,9 @@ enum mp_input_section_flags {
 struct input_ctx;
 struct mp_log;
 
+// Arbitrary upper bound for sanity.
+#define MP_CMD_MAX_ARGS 100
```

Comment:
So this is enforced in `parse_cmd_str`? Should there be some warning when it is hit? Am I correct in understanding this limit can be circumvented with `mp_input_parse_cmd_strv`?

Reply:
The parser will show the same error as before when the maximum number of args was reached. The strv variant is not limited by this, but yeah maybe that does not make much sense.

I think I kept MP_CMD_MAX_ARGS only because javascript.c uses it for some reason, and I was too lazy to update that.

Reply:
> I think I kept MP_CMD_MAX_ARGS only because javascript.c uses it for some reason, and I was too lazy to update that.

Isn't it the correct usage, at least before this patch? If not, what would have been the correct usage? `lua.c` hardcodes it to 50, but it won't work with more than MP_CMD_MAX_ARGS iirc.

Reply:
For some reason, @wm4 hates this question. I'll just merge this for now. We can address eliminating or making more consistent use of `MP_CMD_MAX_ARGS` in a future commit.